src/main/java/com/softserve/teamproject/service/SecurityService.java

```diff
@@ -0,0 +1,7 @@
+package com.softserve.teamproject.service;
+
+public interface SecurityService {
+
+    String findLoggedInUsername();
```

Comment:
Also here should be 2 spaces before method definition, if get your code convention right.